css/about-us.css

```diff
@@ -2,76 +2,367 @@ html {
     font-weight: 400;
     font-style: normal;
-    line-height: 4.085rem;
+    line-height: 2.7rem;
     color: black;
 }
-@media only screen and (max-width: 800px) {
-    html {
-        font-size: 50%;
+.about-us-middle-section {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    background-color: #5C6E93;
+    width: 100%;
+    height: 18.5rem;
+}
+.quote-container {
+    position: relative;
+    display: flex;
+    align-items: center;
+    width: 50%;
+    height: 100%;
+}
+.quotation-text {
+    margin: 0 auto;
+    width: 100%;
+    text-align: left;
+    color: #FFFFFF;
+    font-size: 2.3rem;
+    font-style: normal;
+    font-family: 'Open Sans', sans-serif;
+    line-height: 3.1rem;
+}
+.quotation-mark-left {
+    margin-left: 1.0rem;
+    position: relative;
+    top: -8.0rem;
+    width: 11rem;
+    height: 8.5rem;
+}
+.quotation-mark-right {
+    margin-right: 2.0rem;
+    position: relative;
+    top: 8.0rem;
+    width: 11rem;
+    height: 8.5rem;
+}
+.video-container {
+    position: relative;
+    width: 50%;
+    height: 38.5rem;
+}
+#video-screenshot {
+    width: 63.2rem;
+    height: 100%;
+}
+.text-why-we-here {
+    width: 60%;
+    padding-top: 17.5rem;
+    padding-bottom: 10rem;
+    margin: 0 auto;
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 600;
+    font-style: normal;
+    font-size: 2.6rem;
+    line-height: 3.5rem;
+    text-align: center;
+}
+.about-us-bottom-section {
+    background-image: url(../images/about\ us\ info.png);
+    background-size: 100% 100%;
+    display: flex;
+    align-items: center;
+    margin: 0 auto;
+    width: 60%;
+    height: 50rem;
+}
+.image-text-container {
+    width: 45%;
+    padding-left: 2.5rem;
+    display: flex;
+    flex-direction: column;
+    align-items: left;
+    justify-content: space-around;
+}
+.image-text {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 2.0rem;
+    font-weight: 600;
+    font-style: normal;
+    line-height: 2.7rem;
+    margin-bottom: 4.5rem;
+}
+
+@media only screen and (max-width: 1300px) {
+    .about-us-text-box {
+        margin-bottom: 10rem;
+    }
+    .about-us-middle-section {
+        background-color: transparent;
+        flex-direction: column;
+        height: auto;
+    }
+    .quote-container {
+        width: 100%;
+        margin: 0 auto;
+        background-color: #5C6E93;
+        height: 15rem;
+    }
+    .video-container {
+        margin-top: 7rem;
+        width: 40%;
+        height: 30rem;
+    } 
+    .quotation-text {
+        text-align: center;
+    }
+    .quotation-mark-left {
+        width: 10rem;
+        height: 7rem;
+        top: -7.5rem;
+        margin-left: 15rem;
+    }
+    .quotation-mark-right {
+        width: 10rem;
+        height: 7rem;
+        top: 7.5rem;
+        margin-right: 15rem;
+    }
+    .text-why-we-here {
+        width: 70%;
+        padding-top: 10rem;
+        padding-bottom: 10rem;
```

Comment:
can use a % so you don't need to update the rem for the smaller screen sizes in the queries below because the % will shrink the padding appropriately relative to the screen size. after finding a % that looks good/works, you can delete the padding-top/bottom updates for .text-why-we-here in the smaller screen size queries